src/App.tsx

```diff
@@ -1,61 +1,99 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [firstOptionDisabled, setFirstOptionDisabled] = useState(false);
+  const [formInputs, setFormInputs] = useState({
+    title: '',
+    userId: 0,
+  });
+  const [titleErrorMessage, setTitleErrorMessage] = useState('');
+  const [userErrorMessage, setUserErrorMessage] = useState('');
```

Comment:
imagine if you have form with 20 fields, which you need to validate, will you initiate 20 states for them? maybe better create one state which will return boolean depending on any error existing and then if this state returns `true` check where is this error?